Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
```

Comment:
Are docker files required for local docs development or for Travis? If not, I think they should be removed.

Reply:
Docker is only an option that is by the docusaurus template. It's deleted now.